### PR TITLE
{lib}[GCC/10.2.0] SentencePiece v0.1.95 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)
Depends on:
- [x] #13173

### CODE REVIEW

**Comment by robert-mijakovic** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/s/SentencePiece/SentencePiece-0.1.95-GCC-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,65 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'Bundle'
+
+name = 'SentencePiece'
+version = '0.1.95'
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/s/SentencePiece/SentencePiece-0.1.95-GCC-10.2.0.eb:7]:
```suggestion
version = '0.1.96'
```

**Review by SebastianAchilles — COMMENTED** [review]
SentencePiece v0.1.96 was release. Does it make sense to update?

File: easybuild/easyconfigs/s/SentencePiece/SentencePiece-0.1.95-GCC-10.2.0.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/s/SentencePiece/SentencePiece-0.1.95-GCC-10.2.0.eb:7]: @robert-mijakovic any thoughts on the suggestion for a minor version update?

Comment (robert-mijakovic) [reply easybuild/easyconfigs/s/SentencePiece/SentencePiece-0.1.95-GCC-10.2.0.eb]: @verdurin Sorry for the delay. Done!

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by robert-mijakovic** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/ba5f3d4469682f75b9ef72977e97ad01 for a full test report.

**Comment by robert-mijakovic** [comment]
@verdurin
```
    rpm: /mnt/ds918plus/eb/nuc-fedora32/software/XZ/5.2.5-GCCcore-10.2.0/lib/liblzma.so.5: version `XZ_5.2' not found (required by /lib64/librpmio.so.9)
    /usr/bin/pkg-config: line 8: /usr/bin/-pkg-config: No such file or directory
    Failed to find sentencepiece pkg-config
    Running setup.py install for sentencepiece: finished with status 'error'
```

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/811550aa0d498ef0b9867e657fa891b7 for a full test report.

**Comment by verdurin** [comment]
@robert-mijakovic yes, I think that's because the build node is running Fedora. It built fine on a CentOS 7 node.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @robert-mijakovic!

**Pull request merged**